angular-trello/src/app/models/task.model.ts

```diff
@@ -0,0 +1,7 @@
+export class Task {
```

Comment:
Lakse ti je da koristis interfejse za modele, pogotovo sa "plain" objekte. (Malo je kontraintuitivno ali interfejsi u Typescriptu mogu sadrzati obicna polja). To ti omogucava da umesto sa `new` nove objekte pravis sa `{}` notacijom, a i proizvodi manji bundle jer se interfejsi ne prevode.